src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -4,7 +4,7 @@
     "System.Collections": "4.0.10",
     "System.Linq": "4.0.0",
     "System.Net.Http": "4.0.1-rc3-23809",
-    "System.Net.Primitives": "4.0.11-rc3-23809",
+    "System.Net.Primitives": "4.0.10.0",
```

Comment:
I think I did it because that is how they are in the generations.json file
"System.Net.Primitives": "4.0.10.0",

Looks like the build system doesn't seem to care if the last ".0" is there or not.